src/containerapp/azext_containerapp/custom.py

```diff
@@ -2366,7 +2366,7 @@ def enable_dapr(cmd, name, resource_group_name,
     if 'configuration' not in containerapp_def['properties']:
         containerapp_def['properties']['configuration'] = {}
 
-    if 'dapr' not in containerapp_def['properties']['configuration']:
+    if not containerapp_def['properties']['configuration'].get('dapr', None):
         containerapp_def['properties']['configuration']['dapr'] = {}
```

Comment:
Add test in https://github.com/Azure/azure-cli-extensions/blob/main/src/containerapp/azext_containerapp/tests/latest/test_containerapp_commands.py#L583 and follow the pattern in existing test

Reply:
I have added an additional e2e test for this specific scenario. Is there any documentation explaining how to generate the recordings (possibly for a single test). I had to fiddle with [scripts/ci/test_source.py](scripts/ci/test_source.py) to only run the new test method and store the recording. There must be an easier way, but this worked 🙂 

When you get a chance, can you take another look at this PR?